.gitignore

```diff
@@ -0,0 +1,2 @@
+
+online_store/gt_sport_v1.html
```

Comment:
If you want to remove the gt_sport_v1 files i would suggest just deleting them and committing that change. You can always bring them back later from the git history.

Reply:
Deleted.